src/test/java/htsjdk/samtools/seekablestream/SeekableBufferedStreamTest.java

```diff
@@ -123,6 +124,43 @@ public void testSkip() throws IOException {
         }
     }
 
+    @Test
+    public void testSeek() throws IOException {
+        int bufferSize = 20000;
+        int startPosition = 250000;
+        int length = 5000;
+
+        final int[] RELATIVE_SEEK_OFFSET = new int[]{-bufferSize*2, -bufferSize, -bufferSize/2, -length, -length/2, 0,
+                length/2, length, bufferSize/2, bufferSize, bufferSize*2};
+
+        for (final int seekOffset : RELATIVE_SEEK_OFFSET) {
+            byte[] buffer1 = new byte[length];
+            SeekableStream unBufferedStream = new SeekableFileStream(BAM_FILE);
```

Comment:
You should pull the stream creation into a try-with-resources block so they get closed.

Reply:
Done.